Spigot-Server-Patches/0576-Optimize-Villagers.patch

```diff
@@ -93,25 +111,40 @@ index 35eb3a5a61145e94d5b0c77c0eb13bfa46fac23b..6861b1a345496a83900b0ef702ba3431
 +                }
 +            }
 +        }
-+        // Clean up - vanilla does this only when it runs out, but that would push it to try farther POI's...
-+        this.e.long2LongEntrySet().removeIf((entry) -> entry.getLongValue() < this.d);
++
+         this.g.long2ObjectEntrySet().removeIf((entry) -> {
+-            return !((BehaviorFindPosition.a) entry.getValue()).b(i);
++            return entry.getValue().b < f;
+         });
 +        /*
          Predicate<BlockPosition> predicate = (blockposition) -> {
-             long j = blockposition.asLong();
+             BehaviorFindPosition.a behaviorfindposition_a = (BehaviorFindPosition.a) this.g.get(blockposition.asLong());
```

Comment:
`this.g` is what? Obfhelper. I know this isn't modified, but it will help when updating the code.

Reply:
This is mojang's original code. I doubt you need something changed here.